app/scripts/modules/netflix/tableau/application/appTableau.controller.ts

```diff
@@ -0,0 +1,30 @@
+import {IComponentController, ISCEService, module} from 'angular';
+
+import {AUTHENTICATION_SERVICE, AuthenticationService} from 'core/authentication/authentication.service';
+import {NetflixSettings} from '../../netflix.settings';
+import {Application} from 'core/application/application.model';
+
+require('../tableau.less');
+
+class ApplicationTableauController implements IComponentController {
+
+  public srcUrl: string;
+
+  static get $inject(): string[] {
+    return ['$sce', 'app', 'authenticationService'];
+  }
+
+  constructor(private $sce: ISCEService,
+              private app: Application,
+              private authenticationService: AuthenticationService) {
+
+    const user: string[] = this.authenticationService.getAuthenticatedUser().name.split('@');
+    const url =
+      NetflixSettings.tableau.appSourceUrl.replace('${app}', this.app.name).replace('${user}', user[0]);
+    this.srcUrl = this.$sce.trustAsResourceUrl(url);
+  }
+}
+
+export const TABLEAU_APPLICATION_CONTROLLER = 'spinnaker.netflix.tableau.application.controller';
+module(TABLEAU_APPLICATION_CONTROLLER, [AUTHENTICATION_SERVICE])
+  .controller('AppTableauCtrl', ApplicationTableauController);
```

Comment:
Keep name ordering the same; either it is `TABLEAU_APPLICATION` and `TableauApp` or the inverse. Probably the inverse.